nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/SelectHiveQL.java

```diff
@@ -90,11 +98,59 @@
             .name("hive-query")
             .displayName("HiveQL Select Query")
             .description("HiveQL SELECT query to execute")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .expressionLanguageSupported(true)
+            .build();
+
+    public static final PropertyDescriptor HIVEQL_CSV_HEADER = new PropertyDescriptor.Builder()
+            .name("csv-header")
+            .displayName("CSV Header")
+            .description("Include Header in Output")
+            .required(true)
+            .allowableValues("true", "false")
+            .defaultValue("true")
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor HIVEQL_CSV_ALT_HEADER = new PropertyDescriptor.Builder()
+            .name("csv-alt-header")
+            .displayName("Alternate CSV Header")
+            .description("Comma separated list of header fields")
```

Comment:
Makes sense, I'm thinking we should make that kind of point right in the documentation